setup.py

```diff
@@ -25,6 +25,9 @@
         f"License :: OSI Approved :: {true_licence}",
         "Operating System :: OS Independent",
     ],
+    extras_require={
+        ':sys_platform == "win32"': ["win10toast~=0.9"],
```

Comment:
Should this be added to requirements.txt?

Reply:
Did not realise that you could now - have corrected

Reply:
I also had no idea you could do that. That's...going to be useful elsewhere